packages/website/src/components/Playground.tsx

```diff
@@ -70,6 +70,34 @@ function Playground(): JSX.Element {
   const [position, setPosition] = useState<Monaco.Position | null>(null);
   const [activeTab, setTab] = useState<TabType>('code');
   const [showModal, setShowModal] = useState<TabType | false>(false);
+  const [editorWidth, setEditorWidth] = useState<number | undefined>();
```

Comment:
On page load, this resets to a pretty small default width. It should default to something wider, I think?

<img width="886" alt="Screenshot of the resizeable playground with a very small editor width" src="https://user-images.githubusercontent.com/3335181/190853763-6553451d-d152-4d42-b8c9-44456a4eb712.png">

Reply:
Good catch, I forgot to set `width: 100%` in the CSS module. The next commit should fix this.